nemo/collections/nlp/modules/common/megatron/megatron_transformer_decoder.py

```diff
@@ -130,6 +130,7 @@ def __init__(
             model_type=parent_model_type,
             transformer_block_type=transformer_block_type,
             headscale=headscale,
+            gradient_accumulation_fusion=False,  # TODO: This has to be False for enc-dec models for now.
```

Comment:
Will the model work if grad accumulation is set to true?

Reply:
Yes, this is just a jit fusion thing that was implemented specifically for GPT that I turned off explicitly here for T5.